app/locales/zh-hans.ts

```diff
@@ -0,0 +1,246 @@
+  "ProduceCardChangeUpgrade": "Card Transformation & Upgrade",
+  "ProducePointAddition": "P-Point Increase",
+  "ProducePointGetDisable": "P-Point Obtaining Prohibition",
+  "ProduceItemGetDisable": "P-Items Obtaining Prohibition",
+  "ProduceDrinkGetDisable": "P-Drinks Obtaining Prohibition",
+  "ProduceReward": "Produce Reward",
+  "ProduceRewardSet": "Produce Reward",
+  "AuditionParameterBonusMultiple": "Score Bonus (Percentage)",
+  "LessonSpChangeRatePermilAddition": "SP Lesson Chance Increase (All)",
+  "LessonVocalSpChangeRatePermilAddition": "SP Lesson Chance Increase (Vocal)",
+  "LessonDanceSpChangeRatePermilAddition": "SP Lesson Chance Increase (Dance)",
+  "LessonVisualSpChangeRatePermilAddition": "SP Lesson Chance Increase (Visual)",
+  "LessonSpChangeRatePermilDown": "SP Lesson Chance Decrease (All)",
+  "ExamStatusEnchant": "Lesson/Exam Extra Effect",
+  "LessonPresentProduceCardRewardCountUp": "Card Selectable Options Increase",
+  "LessonPresentProduceCardRewardCountDown": "Card Selectable Options Decrease",
+  "LessonPresentSearchProduceCardRewardCountUp": "LessonPresentSearchProduceCardRewardCountUp", // idk what does this mean
+  "ShopPriceDiscountMultiple": "Shop Price Discount (Percentage)",
+  "ShopProduceCardPriceDiscountMultiple": "Shop Card Price Discount (Percentage)",
+  "ShopProduceDrinkPriceDiscountMultiple": "Shop Drink Price Discount (Percentage)",
+  "ShopProduceCardUpgradePriceDiscountMultiple": "Shop Card Upgrade Price Discount (Percentage)",
+  "ShopProduceCardUpgradePriceSpecify": "Shop Card Upgrade Price Assignment",
+  "ShopProduceCardDeletePriceDiscountMultiple": "Shop Card Deletion Price Discount (Percentage)",
+  "ShopProduceCardDeletePriceSpecify": "Shop Card Deletion Price Assignment",
+  "ProduceResultRewardMoneyUp": "Produce Result Reward Money Increase",
+  "ProduceResultRewardSupportCardEnhancePointUp": "Produce Result Reward S-Pt Increase",
+  "SupportCardEventStaminaRecoverUp": "Event Stamina Recovery Value Increase",
+  "SupportCardEventProducePointAdditionValueUp": "Event Obtained P-Point Increase",
+  "SupportCardEventParameterAdditionValueUp": "Event Increased Parameter Value Increase",
+  "SupportCardEventProbabilityUp": "Event Probability Increase",
+  "LessonPresentProducePointUp": "Lesson Obtained P-Point Increase",
+  "LessonPresentProducePointDown": "Lesson Obtained P-Point Decrease",
+  "SupportCardProduceCardUpgradeProbabilityUp": "Lesson Support Probability Increase",
+  "ProduceCardSelectRerollCountUp": "Card Refreshable Chance Increase",
+  "ProducePointAdditionDisableTrigger": "Initial P-Point Increase",
+  "LessonLimitUp": "Lesson Increasable Parameter Limit Break",
+  // =====================================================
+  // strings underneath this section should not be translated
+  "non-translatables": {
+    language: "语言",
+    "supported-languages": {
+      en: "English",
+      zh-hans: "简体中文",
+    },
+  },
```

Comment:
2. This section is not designed to be translated, that is to say, this section should only be included in `en.ts`. Could you delete this section just like what is stated in the [Localization Instructions](https://github.com/vertesan/hatsuboshi-library#localization-instructions)?